Failing test: Rules Management - Rule Bulk Actions Integration Tests - Serverless Env - Complete Tier.x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action·ts - Rules Management - Rule Bulk Action API @ess @serverless perform_bulk_action @skipInServerlessMKI fill gaps run action should return 400 error when start date is in the future

A test failed on a tracked branch

```
JestAssertionError: expect(received).toContain(expected) // indexOf

Expected substring: "Backfill cannot be scheduled for the future"
Received string:    "Error validating backfill schedule parameters \"{\"start\":\"2025-12-15T11:16:43.159Z\",\"end\":\"2025-12-15T11:16:42.158Z\"}\" - Backfill end must be greater than backfill start"
    at Context.<anonymous> (perform_bulk_action.ts:2903:30)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at Object.apply (wrap_function.js:74:16) {
  matcherResult: {
    message: '\x1B[2mexpect(\x1B[22m\x1B[31mreceived\x1B[39m\x1B[2m).\x1B[22mtoContain\x1B[2m(\x1B[22m\x1B[32mexpected\x1B[39m\x1B[2m) // indexOf\x1B[22m\n' +
      '\n' +
      'Expected substring: \x1B[32m"Backfill cannot be scheduled for the future"\x1B[39m\n' +
      'Received string:    \x1B[31m"Error validating backfill schedule parameters \\"{\\"start\\":\\"2025-12-15T11:16:43.159Z\\",\\"end\\":\\"2025-12-15T11:16:42.158Z\\"}\\" - Backfill end must be greater than backfill start"\x1B[39m',
    pass: false
  }
}
```

First failure: [kibana-on-merge - main](https://buildkite.com/elastic/kibana-on-merge/builds/83861#019b219e-52c3-49cc-baa8-9b003ffca533)

<!-- kibanaCiData = {"failed-test":{"test.class":"Rules Management - Rule Bulk Actions Integration Tests - Serverless Env - Complete Tier.x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action·ts","test.name":"Rules Management - Rule Bulk Action API @ess @serverless perform_bulk_action @skipInServerlessMKI fill gaps run action should return 400 error when start date is in the future","test.failCount":1}} -->

--- a/x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action.ts
+++ b/x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action.ts
@@ -2893,7 +2893,7 @@ export default ({ getService }: FtrProviderContext): void => {
               ids: createdRuleIds,
               action: BulkActionTypeEnum.fill_gaps,
               [BulkActionTypeEnum.fill_gaps]: {
-                start_date: new Date(Date.now() + 1000).toISOString(),
+                start_date: new Date(Date.now() + 10000).toISOString(),
                 end_date: backfillEnd.toISOString(),
               },
             },

